gloo/docs/gloo_routing/virtual_services/security/basic_auth/_index.md

```diff
@@ -4,23 +4,32 @@ weight: 10
 description: Authenticating using a dictionary of usernames and passwords on a virtual service. 
 ---
 
-There are a few different ways to use Gloo External Auth. In certain cases -- such as during testing, or when releasing 
-a new API to a small number of known users -- it may be desirable to secure a set of routes with **Basic Auth**. 
+{{% notice note %}}
+{{< readfile file="static/content/enterprise_only_feature_disclaimer" markdown="true">}}
+{{% /notice %}}
 
-In **BasicAuth**, the Gloo **VirtualService** containing the routes can be configured with a dictionary of 
-authenticated usernames and passwords. When the virtual service configuration changes, Gloo immediately updates the 
-external auth server with the new configuration. On the request path, Envoy asks the external auth service to check 
-the request; any request to a path on that virtual service must have a valid set of credentials or will be denied. 
+{{% notice warning %}}
+{{< readfile file="/static/content/extauth_version_info_note" >}}
+{{% /notice %}}
 
-## Setup
+In certain cases - such as during testing or when releasing a new API to a small number of known users - it may be 
+convenient to secure a Virtual Service using [**Basic Authentication**](https://en.wikipedia.org/wiki/Basic_access_authentication). 
+With this simple authentication mechanism the encoded user credentials are sent along with the request in a standard header.
 
-{{% notice note %}}
-Basic auth is a feature of **Gloo Enterprise**. If you are using Open Source Gloo, this tutorial will not work. 
-{{% /notice %}}
+To secure your VirtualServices using Basic Authentication, you first need to provide Gloo with a set of known users and 
+their passwords. You can then use this information to decide who is allowed to access which routes.
+If a request matches a route on which Basic Authentication is configured, Gloo will verify the credentials in the header 
```

Comment:
"in the header" --> we can be more specific and say which header/the format it will have

Reply:
for those who are truly new to basic auth

Reply:
I wanted to keep the introduction simple and high-level. I feel giving information about the header format is too much detail here. If the reader is interested in the details,  they can click the hyperlink or scroll down (there are details on the header in the last step of the guide).

A good compromise might be giving its name: "in the standard `Authorization` header"

Reply:
i like this compromise